source/_integrations/bosch_alarm.markdown

```diff
@@ -45,19 +45,109 @@ This entity reports state (_disarmed_, _armed_away_, etc.).
 The primary means of authentication for the _Mode 2_ API is the _Automation_ passcode. It needs to be at least 10 characters long, and it is different from the _User_ code -- a shorter numeric pin used to arm/disarm the panel.
 The integration will prompt for the required passcodes, which depend on the panel type.
 
-| Panel | Code |
-| --- | --- |
-| Solution | User [^2] |
+| Panel    | Code       |
+| -------- | ---------- |
+| Solution | User [^2]  |
 | B Series | Automation |
 | G Series | Automation |
-| AMAX | Both |
+| AMAX     | Both       |
 
 [^2]: The user needs to have the "master code functions" authority if you wish to interact with history events.
 
 {% important %}
 Since the _Mode 2_ automation user has "superuser" privileges, it bypasses the regularly configured alarm pin: you will _not_ be prompted for a _User_ code when arming/disarming through the integration.
 {% endimportant %}
 
+## Data updates
+
+The Bosch Alarm Panel {% term integration %} fetches data from the device every 30 seconds.
```

Comment:
```suggestion
The **Bosch Alarm** {% term integration %} fetches data from the device every 30 seconds.
```
I know it is spelled differently further up, but isn't the integration called Bosch Alarm?